platform/darwin/src/MGLStyle.mm

```diff
@@ -254,5 +257,21 @@ - (void)removeStyleClass:(NSString *)styleClass
     }
 }
 
+- (void)setImage:(MGLImage *)image forName:(NSString *)name
+{
+    if (image && name)
```

Comment:
By the way: Do I have to `NSAssert()` now inside both methods because `(MGLImage *)image` and `(NSString *)name` can't be null anyway and as a side quesstion: What is your recommended way to assert for null? 

``` objc
NSAssert([name isEqual:[NSNull null]], "Name cannot be null");

or 

NSAssert(name == NULL, "Name cannot be null");
```